tldextract/tldextract.py

```diff
@@ -39,6 +38,7 @@
 import socket
 import warnings
 
+import idna.codec
```

Comment:
Why not in the alphabetical list above?

Until we get tests to run again, does this complain locally about unused import? Maybe it exports a constant you can pass to `.decode('idna')` below?